consent/sdk_test.go

```diff
@@ -61,8 +61,8 @@ func TestSDK(t *testing.T) {
 		Subject: "subject1",
```

Comment:
Is there a particular reason for updating these tests? Shouldn't they pass as-is?

Reply:
I've updated the PR to make minimal changes to the tests, see explanation in the other thread